app/code/core/Mage/Adminhtml/Block/System/Cache/Edit.php

```diff
@@ -27,13 +27,19 @@
  */
 class Mage_Adminhtml_Block_System_Cache_Edit extends Mage_Adminhtml_Block_Widget
 {
+    /**
+     * Mage_Adminhtml_Block_System_Cache_Edit constructor.
```

Comment:
I've not seen this doc. Is it a requirement from some tools?

Reply:
No, it is not "requirered" ... it is just for complete documention ("no missing docblocks" - e.g. for phpdocumentor, phpdox). I'm pretty sure every __constructor i've touched has this :)